README.md

```diff
@@ -1,2 +1,35 @@
 # odtp-manuals
 All manuals, instructions, documentation and information you need to work with ODTP.
+
+## How to serve this documentation in a docker? 
+
+1. `docker build -t optp-docs .`
+2. `docker run -it --rm --p 8000:8000 odtp-docs`
+
+This will serve the documentation in: http://0.0.0.0:8000/odtp-org/
+
+## Development
+
+### How to serve this documentation in local? 
+
+DISCLAIMER: Still to be properly tested
+
+1. `pip install mkdocs-material`
+2. `brew install cairo freetype libffi libjpeg libpng zlib`
+3. `pip install "mkdocs-material[imaging]"`
+4. `mkdocs new`
+5. `mkdocs serve`
+
+
+### How to create a running development environment?
+
+DISCLAIMER: Still to be properly tested
+
+1. `python -m venv venv`
+2. `source venv/bin/activate`
+3. `pip install mkdocs-material`
+4. `pip install "mkdocs-material[imaging]"`
+
```

Comment:
fix: `mkdocs serve` gave me the following error when I tried to set the package up with a virtual env: (Python 3.11)

```
Traceback (most recent call last):
  File "/Users/smaennel/odtp/odtp-venv/bin/mkdocs", line 8, in <module>
    sys.exit(cli())
             ^^^^^
  File "/Users/smaennel/odtp/odtp-venv/lib/python3.11/site-packages/click/core.py", line 1157, in __call__
    return self.main(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/smaennel/odtp/odtp-venv/lib/python3.11/site-packages/click/core.py", line 1078, in main
    rv = self.invoke(ctx)
         ^^^^^^^^^^^^^^^^
  File "/Users/smaennel/odtp/odtp-venv/lib/python3.11/site-packages/click/core.py", line 1688, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/smaennel/odtp/odtp-venv/lib/python3.11/site-packages/click/core.py", line 1434, in invoke
    return ctx.invoke(self.callback, **ctx.params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/smaennel/odtp/odtp-venv/lib/python3.11/site-packages/click/core.py", line 783, in invoke
    return __callback(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/smaennel/odtp/odtp-venv/lib/python3.11/site-packages/mkdocs/__main__.py", line 270, in serve_command
    serve.serve(**kwargs)
  File "/Users/smaennel/odtp/odtp-venv/lib/python3.11/site-packages/mkdocs/commands/serve.py", line 59, in serve
    config = get_config()
             ^^^^^^^^^^^^
  File "/Users/smaennel/odtp/odtp-venv/lib/python3.11/site-packages/mkdocs/commands/serve.py", line 47, in get_config
    config = load_config(
             ^^^^^^^^^^^^
  File "/Users/smaennel/odtp/odtp-venv/lib/python3.11/site-packages/mkdocs/config/base.py", line 378, in load_config
    errors, warnings = cfg.validate()
                       ^^^^^^^^^^^^^^
  File "/Users/smaennel/odtp/odtp-venv/lib/python3.11/site-packages/mkdocs/config/base.py", line 230, in validate
    run_failed, run_warnings = self._validate()
                               ^^^^^^^^^^^^^^^^
  File "/Users/smaennel/odtp/odtp-venv/lib/python3.11/site-packages/mkdocs/config/base.py", line 188, in _validate
    self[key] = config_option.validate(value)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/smaennel/odtp/odtp-venv/lib/python3.11/site-packages/mkdocs/config/config_options.py", line 182, in validate
    return self.run_validation(value)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/smaennel/odtp/odtp-venv/lib/python3.11/site-packages/mkdocs/config/config_options.py", line 1064, in run_validation
    self.load_plugin_with_namespace(name, cfg)
  File "/Users/smaennel/odtp/odtp-venv/lib/python3.11/site-packages/mkdocs/config/config_options.py", line 1102, in load_plugin_with_namespace
    return (name, self.load_plugin(name, config))
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/smaennel/odtp/odtp-venv/lib/python3.11/site-packages/mkdocs/config/config_options.py", line 1120, in load_plugin
    plugin_cls = self.installed_plugins[name].load()
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/importlib/metadata/__init__.py", line 202, in load
    module = import_module(match.group('module'))
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "<frozen importlib._bootstrap>", line 1206, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1178, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1149, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 690, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/Users/smaennel/odtp/odtp-venv/lib/python3.11/site-packages/material/plugins/social/plugin.py", line 53, in <module>
    from cairosvg import svg2png
  File "/Users/smaennel/odtp/odtp-venv/lib/python3.11/site-packages/cairosvg/__init__.py", line 26, in <module>
    from . import surface  # noqa isort:skip
    ^^^^^^^^^^^^^^^^^^^^^
  File "/Users/smaennel/odtp/odtp-venv/lib/python3.11/site-packages/cairosvg/surface.py", line 9, in <module>
    import cairocffi as cairo
  File "/Users/smaennel/odtp/odtp-venv/lib/python3.11/site-packages/cairocffi/__init__.py", line 47, in <module>
    cairo = dlopen(
            ^^^^^^^
  File "/Users/smaennel/odtp/odtp-venv/lib/python3.11/site-packages/cairocffi/__init__.py", line 44, in dlopen
    raise OSError(error_message)  # pragma: no cover
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
OSError: no library called "cairo-2" was found
no library called "cairo" was found
no library called "libcairo-2" was found
cannot load library 'libcairo.so.2': dlopen(libcairo.so.2, 0x0002): tried: 'libcairo.so.2' (no such file), '/usr/lib/libcairo.so.2' (no such file), '/Users/smaennel/odtp/odtp-manuals/libcairo.so.2' (no such file).  Additionally, ctypes.util.find_library() did not manage to locate a library called 'libcairo.so.2'
cannot load library 'libcairo.2.dylib': dlopen(libcairo.2.dylib, 0x0002): tried: 'libcairo.2.dylib' (no such file), '/usr/lib/libcairo.2.dylib' (no such file), '/Users/smaennel/odtp/odtp-manuals/libcairo.2.dylib' (no such file).  Additionally, ctypes.util.find_library() did not manage to locate a library called 'libcairo.2.dylib'
cannot load library 'libcairo-2.dll': dlopen(libcairo-2.dll, 0x0002): tried: 'libcairo-2.dll' (no such file), '/usr/lib/libcairo-2.dll' (no such file), '/Users/smaennel/odtp/odtp-manuals/libcairo-2.dll' (no such file).  Additionally, ctypes.util.find_library() did not manage to locate a library called 'libcairo-2.dll'
```